packages/contract/lib/contract/index.js

```diff
@@ -1,5 +1,144 @@
-module.exports = {
-  constructorMethods: require("./constructorMethods"),
-  bootstrap: require("./bootstrap"),
-  properties: require("./properties")
-};
+const debug = require("debug")("contract:contract"); // eslint-disable-line no-unused-vars
+let Web3 = require("web3");
+const webUtils = require("web3-utils");
+const execute = require("../execute");
+const bootstrap = require("./bootstrap");
+const constructorMethods = require("./constructorMethods");
+const properties = require("./properties");
```

Comment:
So we're not exporting these anymore... might that be a breaking change?

Reply:
Piggybacking off this, since we maintain this pkg as something we expect at least _some_ users to use independently, it seems like we should make sure breaking changes are more meaningful.

Reply:
Correct me if I'm wrong but I don't think those libraries were externally exposed before. I don't think they are currently accessible, take a look.